examples/rbd/README.md

```diff
@@ -0,0 +1,40 @@
+# How to Use it?
+Install Ceph on the Kubernetes host. For example, on Fedora 21
+
+    # yum -y install ceph
+   
+If you don't have a Ceph cluster, you can set up a [containerized Ceph cluster](https://github.com/rootfs/docker-ceph)
+   
+Then get the keyring from the Ceph cluster and copy it to */etc/ceph/keyring*.
```

Comment:
The key can also be passed to the 'rbd map ...' command like: 'rbd -m <moniplist> --key <secret> map <imagename> ...'